pkg/controllers/podgroup/pg_controller_handler.go

```diff
@@ -61,6 +63,26 @@ func (pg *pgcontroller) addPod(obj interface{}) {
 	pg.queue.Add(req)
 }
 
+func (pg *pgcontroller) addReplicaSet(obj interface{}) {
```

Comment:
@waiterQ As far as I know, when replicaset was created, it would always be `0` replica. And after creating, the replicaset would scale up to defined replica numbers. So why deleting podgroup on both `addReplicaSet` and `updateReplicaSet`, but not only `updateReplicaSet`?

Reply:
yes, you're right. In normal process with one version, volcano just need `updateReplicaSet`, and if consider the situation upgrade from a version to another, there isn't `addReplicaSet` help to cleanup stock podgroups in cluster. `addReplicaSet` is work with already-exist podgroups, `addReplicaSet` work with upcoming podgroups.